config/defaults.yaml

```diff
@@ -1,15 +1,15 @@
-# This configuration file contains configuration values that should apply to all
-# builds of this pathogen repo.
-# Some required keys are missing and must be additionally defined through
-# Snakemake's --configfile/--config options. Those options can also override
-# any values in this file.
+# This configuration file contains all required configuration entries with
+# hmpxv1-focused default values.
+# Use Snakemake's --configfile/--config options to override default values.
 
 exclude: "config/exclude_accessions_mpxv.txt"
 reference: "config/reference.fasta"
 genemap: "config/genemap.gff"
 genbank_reference: "config/reference.gb"
+colors: "config/colors_hmpxv1.tsv"
```

Comment:
_nitpick_

I can't find where this config is being accessed in the workflow.
Maybe this should have been removed as part of https://github.com/nextstrain/monkeypox/pull/109?

Reply:
Hmm, I don't see any references either. I'll make another PR to remove it.

Reply:
#174